apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -68,9 +115,22 @@ const OutOfOfficePage = () => {
             setCurrentlyEditingOutOfOfficeEntry(null);
           }}
           currentlyEditingOutOfOfficeEntry={currentlyEditingOutOfOfficeEntry}
+          oooType={oooType}
+          setTeamOOOEntriesUpdated={setTeamOOOEntriesUpdated}
+          setMyOOOEntriesUpdated={setMyOOOEntriesUpdated}
+        />
+      )}
+      {oooType === "team" ? (
+        <OutOfOfficeEntriesListForTeam
+          editOutOfOfficeEntry={editOutOfOfficeEntry}
+          teamOOOEntriesUpdated={teamOOOEntriesUpdated}
+        />
+      ) : (
+        <OutOfOfficeEntriesList
+          editOutOfOfficeEntry={editOutOfOfficeEntry}
+          myOOOEntriesUpdated={myOOOEntriesUpdated}
```

Comment:
Why are there separate components for Team and Mine view?

As far as I can tell both are same with same components and code?

Reply:
they are similar, but differ in columns displayed. Wanted to avoid having many conditions within same file for team and mine and keep it separated , s

Reply:
handled in single component